p2p/src/authenticated/types.rs

```diff
@@ -95,72 +94,27 @@ pub struct BitVec {
     pub index: u64,
 
     /// The bit vector itself.
-    pub bits: vec::BitVec<u8, Lsb0>,
+    pub bits: BV,
```

Comment:
nit: prefer BitVec

Reply:
`BV` is an alias since this is a variable inside of something already called `BitVec` locally. So I can either do a different alias (`UtilsBitVec`) or fully-qualified `commonware_utils::BitVec`

Reply:
Ah, my b. Disregard.